<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopBookDialog</class>
 <widget class="QWidget" name="TopBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>271</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comfortaa</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>top</string>
     </property>
    </widget>
    <widget class="QPushButton" name="statePageButton">
     <property name="geometry">
      <rect>
       <x>259</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comfortaa</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
</string>
     </property>
     <property name="text">
      <string>top by state</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comfortaa</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter the year for which you
 want to get the top 10 books:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="yearTopEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="topButton2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comfortaa</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>top</string>
     </property>
    </widget>
    <widget class="QPushButton" name="yearPageButton">
     <property name="geometry">
      <rect>
       <x>239</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comfortaa</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
</string>
     </property>
     <property name="text">
      <string>top by year</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comfortaa</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Chose the state what you
wanted to top 10-books:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comfortaa</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Avaliable</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Checked out</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
